package.json

```diff
@@ -342,14 +338,16 @@
   },
   "dependencies": {
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
+    "@dnd-kit/core": "^6.0.3",
+    "@dnd-kit/sortable": "^7.0.0",
```

Comment:
thx for moving these to the correct section but please pin exact version numbers

Reply:
Note: These are actually new dependencies. I've moved from https://github.com/clauderic/react-sortable-hoc, which is no longer maintained, and will stop working with a future version of React to https://github.com/clauderic/dnd-kit, which is the recommended successor.